### PR TITLE
Update NPM dependencies

### PR DESCRIPTION
**Pull request by willeastcott**
Also include lint fixes as a result of the update.

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/framework/components/camera/component.js [hunk 1]
```diff
@@ -148,11 +148,11 @@ class CameraComponent extends Component {
...
      * material.onUpdateShader = function (options) {
-     *    if (options.pass === shaderPassId) {
-     *        options.litOptions.normalMapEnabled = false;
-     *        options.litOptions.useSpecular = false;
-     *    }
-     *    return options;
+     * if (options.pass === shaderPassId) {
+     * options.litOptions.normalMapEnabled = false;
+     * options.litOptions.useSpecular = false;
+     * }
```
Comment (mvaligursky) [inline src/framework/components/camera/component.js:154]: is this required? It does not look as good as before.

File: src/scene/graph-node.js [hunk 2]
```diff
@@ -912,7 +912,7 @@ class GraphNode extends EventHandler {
     /**
      * Returns cached value of negative scale of the world transform.
      *
-     * @returns {number} -1 if world transform has negative scale, 1 otherwise.
+     * @returns {number} - 1 if world transform has negative scale, 1 otherwise.
```
Comment (mvaligursky) [inline src/scene/graph-node.js:915]: hmm it should be '-1' here, not '- 1'.

File: src/framework/components/camera/component.js (hunk 1 above)
Comment (willeastcott) [reply src/framework/components/camera/component.js:154]: I think this is a bug in eslint_plugin_jsdoc. I'll investigate.

File: src/scene/graph-node.js (hunk 2 above)
Comment (willeastcott) [reply src/scene/graph-node.js]: Fixed.

File: src/framework/components/camera/component.js (hunk 1 above)
Comment (mvaligursky) [reply src/framework/components/camera/component.js:154]: this has been fixed / avoided in https://github.com/playcanvas/engine/pull/5472

**Pull request closed without merging**